src/js/components/Drop/DropContainer.js

```diff
@@ -292,7 +301,7 @@ export class DropContainer extends Component {
 
     return (
       <FocusedContainer>
-        <Keyboard onEsc={onEsc} onKeyDown={onKeyDown} target="document">
```

Comment:
It doesn't seem right to remove the document target here. Doesn't that prevent Esc from closing the drop when the drop might not have focus? For example, if Drop has `onClickOutside={false}` but wants Esc to close the drop.

Reply:
correct let me take a look

Reply:
I'm a bit lost at what we're doing here. It seems rather convoluted. Can you include some comments explaining it?